.github/workflows/docs.yml

```diff
@@ -4,6 +4,8 @@ on:
   workflow_dispatch:
   release:
     types: [created]
+  schedule:
+    - cron: "0 */12 * * *" # Every 12 AM/PM every day
```

Comment:
If the article publish once a week, why don't we set the schedule to run once a week. I suggest something like this `0 0 * * 1` or similar.

Reply:
Changed it to every 9AM on every Wednesday